submissions/meduna oluwafemi/HTML/css/productstyle.css

```diff
@@ -0,0 +1,295 @@
+@import url('https://fonts.googleapis.com/css?family=Indie+Flower|Poppins&display=swap');
+
+
+
+* {
+    margin: 0;
+    padding: 0%;
+    font-family:'Poppins', sans-serif; 
+    box-sizing: border-box;
+    text-decoration: none;
+
+}
+
+
```

Comment:
Same comment as the previous one. Ensure you do not have unnecessary line spacing in your files.